File: src/main/java/unificado/core/service/model/Negocio.java
package unificado.core.service.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import lombok.*;

@Data
@Entity
@Builder
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(callSuper = true)
public class Negocio extends Auditable {

    @Id
    @Column(name = "neg_id")
    private Integer negocioId;

    @Column(name = "neg_nombre")
    private String nombre;

    @Column(name = "neg_ip")
    private String ipAddress;

    @Column(name = "neg_db")
    private String database;

    @Column(name = "neg_user")
    private String user;

    private Byte backup;
    private String backendServer;
    private String backendPort;

}

File: src/main/java/unificado/api/rest/kotlin/model/Negocio.kt
package unificado.api.rest.kotlin.model

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table
data class Negocio(

    @Id
    @Column(name = "neg_id")
    var negocioId: Int? = null,

    @Column(name = "neg_nombre")
    var nombre: String = "",

    @Column(name = "neg_ip")
    var ipAddress: String? = null,

    @Column(name = "neg_db")
    var database: String? = null,

    @Column(name = "neg_user")
    var user: String? = null,

    var backup: Byte = 0,
    var backendServer: String = "",
    var backendPort: String = ""

) : Auditable()
